stingray/crossspectrum.py

```diff
@@ -121,6 +130,17 @@ def _make_crossspectrum(self, lc1, lc2):
         if not isinstance(lc2, lightcurve.Lightcurve):
             raise TypeError("lc2 must be a lightcurve.Lightcurve object")
 
+        # Then check that GTIs make sense
+        if self.gti is None:
+            self.gti = cross_two_gtis(lc1.gti, lc2.gti)
+
+        if self.gti.shape[0] != 1:
```

Comment:
This will fail if the user doesn't put in an array of the form `[[0,1]]`, but just `[0,1]`, which I suspect people might do without thinking. I don't know whether there's a good solution to that problem (I've come across it several times), but I thought I'd point it out ...